Generator/Tests/DecodingTests.swift

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2025 the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Testing
+import Yams
+
+@testable import Generator
+
+struct DecodingTests {
+    @Test func decodeDeprecated() throws {
```

Comment:
Having all of these cases in a single test method could lead so some hard to understand failure messages if let's say only one of the cases failed. How about splitting each up into its own test?

Reply:
Ah, for sure. I've changed in this commit: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/19/commits/50522c9796611ec3863501599d938eb725087da4